go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/shurcooL/sanitized_anchor_name
+
+go 1.9
```

Comment:
I don't know what version you officially support, but I tested with 1.9.7 and everything appeared to work. 1.10 is the last supported version of go, so maybe you'd rather I bump this to that?

Reply:
As far as I know it's not documented anywhere, but running `go mod init` creates it with whatever version of Go you're currently using.

Reply:
I see, thanks. Let's leave it out for now, and consider adding it when we have a better understanding of it.

Reply:
Since it's put in by the Go tool by default, shouldn't we leave it in and not remove it unless we understand why? It makes me nervous to do things that aren't the default thing recommended by the tooling.

Reply:
What version of Go did you use? When I do `go mod init` with an explicit module path, I don't get a `go` directive in it. See:

```
$ go version
go version go1.11.2 darwin/amd64
$ go mod init github.com/shurcooL/sanitized_anchor_name
go: creating new go.mod: module github.com/shurcooL/sanitized_anchor_name
$ ls
go.mod
$ cat go.mod 
module github.com/shurcooL/sanitized_anchor_name
```

Also, examples of go.mod files without a `go` directive are shown at https://golang.org/cmd/go/#hdr-The_go_mod_file, https://research.swtch.com/vgo-module, and https://github.com/golang/go/wiki/Modules.

Reply:
Oops, replied in the wrong place apparently. I ended up finding documentation for this here: https://tip.golang.org/cmd/go/#hdr-The_go_mod_file

So presumably it's still only in 1.12

Reply:
Thanks. There's not a lot of information there to give me an understanding of the implications of using that directive and its different values.

There's more information about it at https://github.com/golang/go/issues/28221#issuecomment-432765035.

I prefer not to add a `go` directive to `go.mod` for now. I will consider adding it later when it's clear what the benefits and consequences are.

Go 1.11.4 is the current stable Go release, and running `go mod init` with it produces a `go.mod` file with only the `module` directive. Go 1.12 is not yet released, it's in beta, so in theory anything there can change before the final release. I don't expect it will, my point is that it's clearly valid to rely on a `go.mod` generated by 1.11.4.

So, please remove the `go` directive and I will be able to merge this PR. Thanks.